modules/windows-node-services.adoc

```diff
@@ -22,6 +22,9 @@ The following Windows-specific services are installed on each Windows node:
 |Windows Instance Config Daemon (WICD)
 |Creates the Windows worker node. Starts the containerd container runtime and the kubelet on the node. Maintains the state of Windows Services on the node using a config map created by the Windows Machine Config Operator (WMCO).
 
+|Windows Machine Config Bootstrapper (WMCB)
```

Comment:
Is there a reason to add WMCB back?

Reply:
@aravindhp I didn't remove the WMCB. Apparently I misunderstood what [you said in Slack](https://coreos.slack.com/archives/CM4ERHBJS/p1668542947100979?thread_ts=1668461867.424479&cid=CM4ERHBJS). And, you left WMCB in [your presentation diagrams](https://docs.google.com/presentation/d/1KKjXcxaqUD_2JBnAjwkEv1po2VFGkMgoqHNcyMYBGjs/edit#slide=id.gbd5f5dd60f_0_0).

Reply:
Sorry there are multiple diagrams that need to be updated in that deck. [Here](https://docs.google.com/presentation/d/1KKjXcxaqUD_2JBnAjwkEv1po2VFGkMgoqHNcyMYBGjs/edit#slide=id.g18e22cfe629_0_0) is the one that I updated.

Reply:
>Apparently I misunderstood what [you said in Slack](https://coreos.slack.com/archives/CM4ERHBJS/p1668542947100979?thread_ts=1668461867.424479&cid=CM4ERHBJS).

Sorry I should have clarified that the WICD work not being fully complete in 7.0.0 only implies WMCO still does some of the work WICD will do in the future. All WMCB responsibilities have been removed in 7.0.0.